release/release_logs/1.11.0/stress_tests/stress_test_placement_group.json

```diff
@@ -1,9 +1,9 @@
 {
   "success": 1,
-  "avg_pg_create_time_ms": 0.6551887567598629,
-  "avg_pg_remove_time_ms": 3.2382229564551754,
-  "_runtime": 294.8811585903168,
-  "_session_url": "https://beta.anyscale.com/o/anyscale-internal/projects/prj_2xR6uT6t7jJuu1aCwWMsle/clusters/ses_JrxFXTCkW1yA9mnRZKTayGEe",
-  "_commit_url": "https://s3-us-west-2.amazonaws.com/ray-wheels/releases/1.10.0/5ea565317a8104c04ae7892bb9bb41c6d72f12df/ray-1.10.0-cp37-cp37m-manylinux2014_x86_64.whl",
+  "avg_pg_create_time_ms": 1.246994148648686,
```

Comment:
Is this regression or noise? @rkooo567

Reply:
<img width="1440" alt="Screen Shot 2022-02-01 at 6 20 49 PM" src="https://user-images.githubusercontent.com/18510752/152083377-f90a3a06-cdf2-4a80-8fdc-c465fb58381a.png">
^ Lower is better

I think it is noise. I found the variance is pretty big (0.7~1.1). Do you mind running this one more time?

Reply:
Reran and it was 1.15: https://buildkite.com/ray-project/periodic-ci/builds/2610#5932b008-a14f-46a5-91b7-654bc5b7141d/113-388